Signal/src/view controllers/SignalAttachment.swift

```diff
@@ -5,7 +5,7 @@
 import Foundation
 import MobileCoreServices
 
-enum SignalAttachmentError: String {
+enum SignalAttachmentError: Int {
```

Comment:
nit: no need for `Int` either, unless you intend to use the `.rawValue` somewhere. You should be able to just `enum SignalAttachmentError {`

But (see my other comment), I think we should `enum SignalAttachmentError: LocalizedError {`